manifests/p/PostgreSQL/pgAdmin/7.0/PostgreSQL.pgAdmin.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: PostgreSQL.pgAdmin
+PackageVersion: 7.0
+PackageLocale: en-US
+Publisher: The pgAdmin Development Team
+PublisherUrl: https://www.postgresql.org
+PublisherSupportUrl: https://www.postgresql.org/support
+PrivacyUrl: https://www.postgresql.org/about/policies/privacy
+Author: The pgAdmin Development Team
+PackageName: pgAdmin 4
+PackageUrl: https://www.pgadmin.org
+License: The PostgreSQL Licence
+LicenseUrl: https://www.pgadmin.org/licence
+Copyright: Copyright © 1996-2023 The PostgreSQL Global Development Group
+CopyrightUrl: https://www.postgresql.org/about/policies/trademarks
+ShortDescription: Open Source administration and development platform for PostgreSQL
+Description: |-
+  pgAdmin is designed to answer the needs of all users, from writing simple SQL queries to developing complex databases.
+  The graphical interface supports all PostgreSQL features and makes administration easy.
+  The application also includes a syntax highlighting SQL editor, a server-side code editor, an SQL/batch/shell job scheduling agent, support for the Slony-I replication engine and much more.
+  Server connection may be made using TCP/IP or Unix Domain Sockets (on *nix platforms), and may be SSL encrypted for security.
+  No additional drivers are required to communicate with the database server.
+Moniker: pgadmin
+Tags:
+- pgsql
+- postgre
+- sql
+ReleaseNotesUrl: https://www.pgadmin.org/docs/pgadmin4/6.21/release_notes_6_21.html

```

```suggestion
ReleaseNotesUrl: https://www.pgadmin.org/docs/pgadmin4/7.0/release_notes_7_0.html
```
Update ReleaseNotesUrl